ebml/EbmlElement.h

```diff
@@ -208,10 +218,13 @@ class EBML_DLL_API EbmlCallbacks {
         inline const EbmlSemanticContext & GetContext() const { return Context; }
         inline const char * GetName() const { return DebugName; }
         inline EbmlElement & NewElement() const { return Create(); }
+        /// is infinite/unknown size allowed
+        inline bool CanInfinite() const { return canInfinite; }
 
     private:
     EbmlElement & (* const Create)();
     const EbmlId & GlobalId;
+    const bool canInfinite;
```

Comment:
Existing member variables use capitalized camel case. Please use it for new variables as well. Then rename the function slightly in order to to clash: `bool CanInfinite;` and `bool CanHaveInfiniteSize() const`. Thanks.

Reply:
Done